geode-management/src/main/java/org/apache/geode/management/api/ClusterManagementService.java

```diff
@@ -66,6 +70,24 @@
   <T extends CacheElement & CorrespondWith<R>, R extends RuntimeInfo> ClusterManagementResult<T, R> get(
       T config);
 
+  /**
+   * Execute a given operation asynchronously.
```

Comment:
How does this relate to the above OperationManager? Was I wrong about OperationManager being async?